### ISSUE TITLE
switch.fluxer not displayed

### ISSUE THREAD

**Issue by robin13** [estevez-dev/ha_client#453]
**HA Client version:** 0.6.6

**Home Assistant version:** 0.98.2

**Device name:** Nexus 5x
**Android version:** 8.1.0

**Connection type:** local IP

**Login type:** HA Login

**Description**
I have added fluxer switch to a view which I can see on the web interface, but not on the app.  Any other elements I add are shown, just not `fluxer.switch` ...

**Screenshots**

What I see in the app:

![Screenshot_20190914-154410_HA_Client](https://user-images.githubusercontent.com/174291/64909001-c5cf8d00-d706-11e9-98cb-31cfe41a5c02.png)

What I see in the web UI:

![screenshot-2019-09-14_15-45-42](https://user-images.githubusercontent.com/174291/64909002-c9631400-d706-11e9-8159-5da853870b33.png)

**Logs**

```
0914 15:44:05 [Debug] :  ----view id: null
0914 15:44:05 [Debug] :  App was previously registered. Checking...
0914 15:44:05 [Debug] :  [Sending] ==> HTTP /api/webhook/ccdae2d7e08e70ede8dc83a243acabdb6ee66c9210ad7a6e2e695f99b1c49bed
0914 15:44:06 [Debug] :  [Received] <== HTTP 200
0914 15:44:06 [Debug] :  App registration works fine
0914 15:44:06 [Debug] :  [Received] <== event.state_changed: device_tracker.cundr_print
0914 15:44:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_current
0914 15:44:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_power
0914 15:44:13 [Debug] :  [Received] <== event.state_changed: sensor.dark_sky_temperature
0914 15:44:13 [Debug] :  [Received] <== event.state_changed: sensor.dark_sky_wind_speed
0914 15:44:15 [Debug] :  AppLifecycleState.inactive
0914 15:44:16 [Debug] :  [Received] <== event.state_changed: sun.sun
0914 15:44:18 [Debug] :  AppLifecycleState.paused
0914 15:44:18 [Debug] :  [Received] <== event.state_changed: sensor.fritz_netmonitor
0914 15:44:22 [Debug] :  [Received] <== event.state_changed: device_tracker.cundr_print
0914 15:44:22 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_01_voltage
0914 15:44:32 [Debug] :  [Received] <== event.state_changed: sensor.es_publish_queue
0914 15:44:38 [Debug] :  [Received] <== event.state_changed: device_tracker.cundr_print
0914 15:44:42 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_01_power
0914 15:44:42 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_01_energy
0914 15:44:49 [Debug] :  [Received] <== event.state_changed: sensor.fritz_netmonitor
0914 15:44:50 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_02_voltage
0914 15:44:50 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_02_energy
0914 15:45:01 [Debug] :  [Received] <== event.state_changed: sensor.time
0914 15:45:04 [Debug] :  [Received] <== event.state_changed: sensor.es_publish_queue
0914 15:45:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_current
0914 15:45:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_power
0914 15:45:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_energy
0914 15:45:20 [Debug] :  [Received] <== event.state_changed: sensor.fritz_netmonitor
0914 15:45:22 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_01_voltage
0914 15:45:26 [Debug] :  [Received] <== event.state_changed: device_tracker.cundr_print
0914 15:45:35 [Debug] :  [Received] <== event.state_changed: sensor.es_publish_queue
0914 15:45:42 [Debug] :  [Received] <== event.state_changed: device_tracker.cundr_print
0914 15:45:42 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_01_power
0914 15:45:42 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_01_energy
0914 15:45:44 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_th10_03_temperature
0914 15:45:51 [Debug] :  [Received] <== event.state_changed: sensor.fritz_netmonitor
0914 15:45:58 [Debug] :  [Received] <== event.state_changed: device_tracker.cundr_print
0914 15:46:01 [Debug] :  [Received] <== event.state_changed: sensor.time
0914 15:46:01 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_th16_04_temperature
0914 15:46:06 [Debug] :  [Received] <== event.state_changed: sensor.es_publish_queue
0914 15:46:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_current
0914 15:46:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_voltage
0914 15:46:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_power
0914 15:46:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_energy
0914 15:46:14 [Debug] :  [Received] <== event.state_changed: device_tracker.cundr_print
0914 15:46:22 [Debug] :  [Received] <== event.state_changed: sensor.fritz_netmonitor
0914 15:46:30 [Debug] :  [Received] <== event.state_changed: device_tracker.cundr_print
0914 15:46:37 [Debug] :  [Received] <== event.state_changed: sensor.es_publish_queue
0914 15:46:37 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_th16_01_temperature
0914 15:46:42 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_01_voltage
0914 15:46:42 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_01_power
0914 15:46:42 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_01_energy
0914 15:46:44 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_th10_03_temperature
0914 15:46:50 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_02_voltage
0914 15:46:50 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_pow_02_energy
0914 15:46:53 [Debug] :  [Received] <== event.state_changed: sensor.fritz_netmonitor
0914 15:47:01 [Debug] :  [Received] <== event.state_changed: sensor.time
0914 15:47:01 [Debug] :  [Received] <== event.state_changed: sensor.sonoff_th16_04_temperature
0914 15:47:02 [Debug] :  [Received] <== event.state_changed: device_tracker.rcl_laptop_lan
0914 15:47:08 [Debug] :  [Received] <== event.state_changed: sensor.es_publish_queue
0914 15:47:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_current
0914 15:47:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_voltage
0914 15:47:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_power
0914 15:47:08 [Debug] :  [Received] <== event.state_changed: sensor.espurna_blitzwolf_02_energy
0914 15:47:19 [Debug] :  AppLifecycleState.inactive
0914 15:47:19 [Debug] :  AppLifecycleState.resumed
0914 15:47:19 [Debug] :  [Sending] ==> {"type":"get_states","id":22}
0914 15:47:19 [Debug] :  [Sending] ==> {"type":"lovelace/config","id":23}
0914 15:47:19 [Debug] :  [Sending] ==> {"type":"get_config","id":24}
0914 15:47:19 [Debug] :  [Sending] ==> {"type":"get_services","id":25}
0914 15:47:19 [Debug] :  [Sending] ==> {"type":"auth/current_user","id":26}
0914 15:47:19 [Debug] :  [Sending] ==> {"type":"get_panels","id":27}
0914 15:47:19 [Debug] :  [Sending] ==> {"type":"subscribe_events","id":28,"event_type":"state_changed"}
0914 15:47:19 [Debug] :  [Received] <== Request id 22 was successful
0914 15:47:19 [Debug] :  [Received] <== Request id 24 was successful
0914 15:47:19 [Debug] :  [Received] <== Request id 27 was successful
0914 15:47:19 [Debug] :  [Received] <== Request id 28 was successful
0914 15:47:19 [Debug] :  [Received] <== Request id 23 was successful
0914 15:47:19 [Debug] :  [Received] <== Request id 25 was successful
0914 15:47:19 [Debug] :  [Received] <== Request id 26 was successful
0914 15:47:19 [Debug] :  Parsing 184 Home Assistant entities
0914 15:47:19 [Debug] :  Got 24 services
0914 15:47:19 [Debug] :  Media extractor: null
0914 15:47:19 [Debug] :  Creating Lovelace UI
0914 15:47:19 [Debug] :  --Title: Home
0914 15:47:19 [Debug] :  --Views count: 9
0914 15:47:19 [Debug] :  ----view id: null
0914 15:47:19 [Debug] :  ----view id: null
0914 15:47:19 [Debug] :  ----view id: null
0914 15:47:19 [Debug] :  ----view id: null
0914 15:47:19 [Debug] :  ----view id: null
0914 15:47:19 [Debug] :  ----view id: null
0914 15:47:19 [Debug] :  ----view id: null
0914 15:47:19 [Debug] :  ----view id: null
0914 15:47:19 [Debug] :  ----view id: null
0914 15:47:19 [Debug] :  App was previously registered. Checking...
0914 15:47:19 [Debug] :  [Sending] ==> HTTP /api/webhook/ccdae2d7e08e70ede8dc83a243acabdb6ee66c9210ad7a6e2e695f99b1c49bed
0914 15:47:20 [Debug] :  [Received] <== HTTP 200
0914 15:47:20 [Debug] :  App registration works fine
```

**Comment by estevez-dev** [comment]
Could you please show a config for this card? Thanks

**Comment by estevez-dev** [comment]
One more thing. If you just added this entity to a card in your web UI, you need to full reload the app. Tap three-dot button in upper-right corner and select "Reload"

**Comment by robin13** [comment]
The config for the card is:

```
  - badges: []
    cards:
      - entities:
          - entity: light.h801_led_01
          - entity: light.h801_led_02
          - entity: light.h801_led_03
          - entity: light.sonoff_b1_01
          - entity: switch.fluxer
          - entity: light.sonoff_b1_01
        show_header_toggle: false
        type: entities
    path: lights
    title: lights
```

I have done the `Reload`, but no change.  If I add / remove another element (here the last one `light.sonoff_b1_01` I immediately see this being added/removed in the mobile app, so I am pretty sure that changes as such _are_ being noticed, it is just the `switch.fluxer` which is not displayed.

**Comment by estevez-dev** [comment]
Should be fixed in 0.7.0
Feel free to reopen if will be actual after 0.7.0 update
